dotnet/src/SemanticKernel.Core/Extensions/KernelSemanticFunctionExtensions.cs

```diff
@@ -175,7 +175,7 @@ public static class KernelSemanticFunctionExtensions
     /// <param name="parentDirectory">Directory containing the plugin directory, e.g. "d:\myAppPlugins"</param>
     /// <param name="pluginDirectoryNames">Name of the directories containing the selected plugins, e.g. "StrategyPlugin"</param>
     /// <returns>A list of all the semantic functions found in the directory, indexed by function name.</returns>
-    public static IDictionary<string, ISKFunction> ImportSemanticPluginFromDirectory(
+    public static IDictionary<string, ISKFunction> ImportSemanticFunctionsFromDirectory(
         this IKernel kernel, string parentDirectory, params string[] pluginDirectoryNames)
```

Comment:
consider/nit: for consistency
```suggestion
        this IKernel kernel, string parentDirectory, params string[] semanticFunctionDirectoryNames)
```

Reply:
Let me check with @matthewbolanos on this. Maybe `ImportSemanticFunctionsFromDirectories` would be better.

Reply:
Will follow up next week on this topic